src/algebra/pointwise.lean

```diff
@@ -860,6 +860,21 @@ le_antisymm
     (λ k hk, subset_closure ⟨1, k, H.one_mem, hk, one_mul k⟩))
   (by conv_rhs { rw [← closure_eq H, ← closure_eq K] }; apply closure_mul_le)
 
+lemma pow_smul_mem_closure_smul {N : Type*} [comm_monoid N] [mul_action M N]
+  [is_scalar_tower M N N] (r : M) (s : set N) (x : closure s) :
+    ∃ n : ℕ, r ^ n • (x : N) ∈ closure (r • s) :=
```

Comment:
It might be a good idea to prove `∃ N, ∀ n ≥ N, ...` right away. An I would use `{x : N} (hx : x ∈ closure s)` instead of `x : closure s`.

Reply:
I tried a bit but I don't think I could prove that right away.
However there should indeed be a lemma in such form. I'm adding it now.

Reply:
Oops. This is probably not true. Since the closure is taken wrt the structure on `N`, it might not be closed under the action of `M`.